app/controllers/comments_controller.rb

```diff
@@ -33,6 +34,9 @@ def create
       if @comment.save
         format.html { redirect_back fallback_location: root_path, notice: "Comment was successfully created." }
         format.json { render :show, status: :created, location: @comment }
+
```

Comment:
Great job linking to the js files. Make sure you are mindful of white spaces and remove any unnecessary ones.